jbang edit --live doesn't support non-single-word commands for the editor

# Scenario

I have Intellij IDEA installed through flatpak on Fedora, so running IDEA is not as simple as running the `idea` command.
Instead the command looks like `/usr/bin/flatpak run --branch=stable --arch=x86_64 --command=idea com.jetbrains.IntelliJ-IDEA-Community`

# Issue

Running `jbang edit --live="/usr/bin/flatpak run --branch=stable --arch=x86_64 --command=idea com.jetbrains.IntelliJ-IDEA-Community" myfile.java` fails with:

```
$ jbang --verbose edit --live="/usr/bin/flatpak run --branch=stable --arch=x86_64 --command=idea com.jetbrains.IntelliJ-IDEA-Community" myfile.java
[jbang] Running `/usr/bin/flatpak run --branch=stable --arch=x86_64 --command=idea com.jetbrains.IntelliJ-IDEA-Community /home/zakkak/.jbang/cache/projects/myfile.java_jbang_e877dd301105098ab7795f78a9be653f505c4bb9733cba607c9ebc54da7aa7d1/tag`
java.io.IOException: Cannot run program "/usr/bin/flatpak run --branch=stable --arch=x86_64 --command=idea com.jetbrains.IntelliJ-IDEA-Community": error=2, No such file or directory
        at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1128)
        at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1071)
        at dev.jbang.cli.Edit.doCall(Edit.java:55)
        at dev.jbang.cli.BaseCommand.call(BaseCommand.java:59)
        at dev.jbang.cli.BaseCommand.call(BaseCommand.java:13)
        at picocli.CommandLine.executeUserObject(CommandLine.java:1933)
        at picocli.CommandLine.access$1100(CommandLine.java:145)
        at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2332)
        at picocli.CommandLine$RunLast.handle(CommandLine.java:2326)
        at picocli.CommandLine$RunLast.handle(CommandLine.java:2291)
        at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2159)
        at picocli.CommandLine.execute(CommandLine.java:2058)
        at dev.jbang.Main.main(Main.java:14)
Caused by: java.io.IOException: error=2, No such file or directory
        at java.base/java.lang.ProcessImpl.forkAndExec(Native Method)
        at java.base/java.lang.ProcessImpl.<init>(ProcessImpl.java:340)
        at java.base/java.lang.ProcessImpl.start(ProcessImpl.java:271)
        at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1107)
        ... 12 more
```

--- a/src/main/java/dev/jbang/cli/Edit.java
+++ b/src/main/java/dev/jbang/cli/Edit.java
@@ -49,7 +49,7 @@ public class Edit extends BaseScriptCommand {
 						+ System.getenv("GITPOD_WORKSPACE_URL") + "#" + project.getAbsolutePath() + "\n\n");
 			} else {
 				List<String> optionList = new ArrayList<>();
-				optionList.add(liveeditor);
+				optionList.addAll(Arrays.asList(liveeditor.split(" ")));
 				optionList.add(project.getAbsolutePath());
 				info("Running `" + String.join(" ", optionList) + "`");
 				Process process = new ProcessBuilder(optionList).start();

